pytorch/inference/buildspec.yml

```diff
@@ -13,8 +13,8 @@ repository_info:
     root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
```

Comment:
is this the same functional result as just commenting out line 8 autopatch_build: "True"?

Reply:
No, disabling `autopatch_build` will only stop it from building an image that as `-autopatch`. 

However, some tests such as `test_package_version_regression_in_image` will run if we provide `latest_release_image`. Usually we don't disable it to make sure that all packages in the released image all exist in the current image. However, sometimes packages will remove some utility packages from their dependency tree causing the test to fail. In certain cases, we don't care about said utility package because it isn't a vital package to DLC so we bypass that test by commenting out the field in buildspec. This is the only case I've hit that we need to comment out but @shantanutrip would know more about other tests that are being run when those fields are declared in the buildspec file.

Reply:
gotcha, but autopatch should still be disabled at the top of this file right?

Reply:
Ah, I see it wasn't commented out. thanks! Editing the code and re-running tests.